### PR TITLE
feat(Breadcrumb): Add aria attributes

### LINKED ISSUE #1060: Add aria attributes to Breadcrumbs component
### Description

Component | Sub element | Attribute | Value
-- | -- | -- | --
Breadcrumbs |   |   |  
  | last element | aria-current | page
  | separator | aria-hidden |

### PR DESCRIPTION
**Pull request by m7kvqbe1**
## Related issue

Closes #1060

## Overview

Add aria attributes to Breadcrumb component.

## Reason

> We want to meet accessibility guidelines for all existing components.

## Work carried out

- [x] Add aria attributes
- [x] Minor refactor
- [x] Add automated tests

## Developer notes

https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html

### CODE REVIEW

File: packages/react-component-library/src/components/Breadcrumbs/Breadcrumbs.tsx [hunk 1]
```diff
@@ -25,7 +25,15 @@ export const Breadcrumbs: React.FC<Nav<BreadcrumbsItemProps>> = ({
     }
   )
 
-  return <ul className={classes}>{mapped}</ul>
+  return (
+    <nav
+      className={classes}
+      aria-label="breadcrumb"
```
Comment (thyhjwb6) [inline packages/react-component-library/src/components/Breadcrumbs/Breadcrumbs.tsx]: Does this need to be `Breadcrumb`?

**Review by thyhjwb6 — COMMENTED** [review]
Looks good. Question about the `aria-label`..

File: packages/react-component-library/src/components/Breadcrumbs/Breadcrumbs.tsx (hunk 1 above)
Comment (m7kvqbe1) [reply packages/react-component-library/src/components/Breadcrumbs/Breadcrumbs.tsx]: I'll capitalize it.

**Review by thyhjwb6 — APPROVED** [review]
👍

**Pull request merged**